backend/api/v1/auth_service.go

```diff
@@ -730,7 +730,7 @@ func validateEmail(email string) error {
 
 const (
 	// issuerName is the name of the issuer of the OTP token.
-	issuerName = "bytebase"
+	issuerName = "Bytebase"
 )
```

Comment:
Any reason why we change it back to Bytebase? Wannt make a comment if needed.

Reply:
My bad, it will show in the title place. So it should be `Bytebase`.

<img width="409" alt="image" src="https://user-images.githubusercontent.com/24653555/223955256-ebac8c47-988e-41a8-8d09-66f571448cea.png">